docs: release notes for 24.7 (#631)
stackabletech/documentation: modules/concepts/pages/container-images.adoc

[#multi-platform-support]
== Multi-platform support

> See xref:concepts:multi-platform-support.adoc[this] page for an overview and the current status.

Reviewer: ```suggestion
See our documentation xref:concepts:multi-platform-support.adoc[on multi-platform support] for an overview and the current status.
```
